sklearn/datasets/_kddcup99.py

```diff
@@ -114,7 +116,8 @@ def fetch_kddcup99(*, subset=None, data_home=None, shuffle=False,
         .. versionadded:: 0.20
     """
     data_home = get_data_home(data_home=data_home)
-    kddcup99 = _fetch_brute_kddcup99(data_home=data_home,
+    kddcup99, feat_cols, target_col = _fetch_brute_kddcup99(
```

Comment:
`feature_names` and `target_names` would be better names

Reply:
DONE.